resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+          </thead>
+        </table>
+      operationId: reportOutcomeOfPropertyPurchase
+      parameters:
+      - $ref: '#/components/parameters/acceptHeader'
+      - $ref: '#/components/parameters/authorizationHeader'
+      - $ref: '#/components/parameters/lisaManagerReferenceNumber'
+      - $ref: '#/components/parameters/accountId'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              allOf:
+              - $ref: '#/components/schemas/lifetimeIsaManagerAccountsEventsPurchaseOutcomesRequest'
+            examples:
+              example-1:
+                value:
+                  fundReleaseId: '3456789001'
+                  eventDate: 2017-10-10
+                  propertyPurchaseResult: Purchase completed
+                  propertyPurchaseValue: 250000
+              example-2:
+                value:
+                  eventDate: 2017-10-05
+                  propertyPurchaseResult: Purchase completed
+                  propertyPurchaseValue: 250000
+                  supersede:
+                    originalLifeEventId: '5678900001'
+                    originalEventDate: 2017-10-10
+              example-3:
+                value:
+                  fundReleaseId: '3456789002'
+                  eventDate: 2017-05-05
+                  propertyPurchaseResult: Purchase failed
+        required: true
+      responses:
+        '201':
+          description: Created
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaManagerAccountsEventsPurchaseOutcomesResponse'
+              examples:
+                lifeEventCreated:
+                  value:
+                    status: 201
+                    success: true
+                    data:
+                      message: Purchase outcome created
+                      lifeEventId: '5678900001'
+                lifeEventSuperseded:
+                  value:
+                    status: 201
+                    success: true
+                    data:
+                      message: Purchase outcome superseded
+                      lifeEventId: '5678900002'
+        '403':
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaErrorResponse'
+              examples:
+                mismatchError:
+                  value:
+                    code: SUPERSEDED_LIFE_EVENT_MISMATCH_ERROR
+                    message: originalLifeEventId and the originalEventDate do not match the information in the original request
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaErrorResponse'
+              examples:
+                accountNotFound:
```

Comment:
Please add missing description, thanks.

                accountNotFound:
                  description: Enter a real accountId
                  value:
                    code: INVESTOR_ACCOUNTID_NOT_FOUND
                    message: The accountId does not match with HMRC’s records